### ISSUE TITLE
Description term elements in show page lack sufficient color contrast

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1069]
Violation: Elements must have sufficient color contrast. Currently at color contrast of 4.47. Needs to meet 4.5.
